### DISCUSSION TITLE
Where can we find allowed `symbols` for password?

### DISCUSSION THREAD

**Discussion by Yberion** [zitadel/zitadel#7916] [Q&A] [answered]
Hello,

- https://zitadel.com/docs/guides/manage/console/default-settings#password-complexity

Is there a place where we can find all allowed symbols for a password? Maybe this is a regex somewhere?

Does a symbol equals anything but number/letter for Zitadel?

**Comment by hifabienne** [comment, marked as answer]
Yes, you can store everything that is not a letter or a number.

**Reply by Yberion** [reply]
Thanks,

I was going to edit my question as I found the answer checking the html of the Zitadel register page.

![image](https://github.com/zitadel/zitadel/assets/4186385/e3d53e4c-acea-4e06-9cf4-f3a0f1153887)

I think this should be added in the documentation, I will probably create a ticket for that.
